<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainform</class>
 <widget class="QDialog" name="mainform">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1028</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Project manager</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item row="5" column="2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Model</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="btn_add_model">
        <property name="text">
         <string>Add model</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_delete_model">
        <property name="text">
         <string>Delete model</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_import_model">
        <property name="text">
         <string>Import model</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_export_model">
        <property name="text">
         <string>Export model</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Project</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="btn_open_project">
        <property name="text">
         <string>Open project</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_new_project">
        <property name="text">
         <string>New project</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_duplicate_project">
        <property name="text">
         <string>Duplicate project</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_delete_project">
        <property name="text">
         <string>Delete project</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_export_project">
        <property name="text">
         <string>Export project</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_import_project">
        <property name="text">
         <string>Import project</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_update_project">
        <property name="text">
         <string>Update project</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="2">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <layout class="QHBoxLayout" name="message_bar_layout"/>
   </item>
   <item row="1" column="2">
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Service</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <widget class="QComboBox" name="service"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QWidget" name="progressbar_widget" native="true"/>
   </item>
   <item row="1" column="0" rowspan="6" colspan="2">
    <widget class="QTreeWidget" name="tree_widget">
     <column>
      <property name="text">
       <string notr="true">Project</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Version</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>SRID</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Workspace</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Chargement des styles</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tree_widget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
